internal/hcsoci/resources_lcow.go

```diff
@@ -35,12 +35,34 @@ func getGPUVHDPath(coi *createOptionsInternal) (string, error) {
 	return gpuVHDPath, nil
 }
 
+func getContainerExtraLayers(annotations map[string]string) ([]string, error) {
+	csExtraLayers, ok := annotations[oci.AnnotationContainerExtraLayers]
+	if !ok || csExtraLayers == "" {
+		return nil, nil
+	}
+	layers := strings.Split(csExtraLayers, ",")
+	for _, l := range layers {
+		if _, err := os.Stat(l); err != nil {
+			return nil, errors.Wrapf(err, "failed to find path to container's extra layers at %s", l)
+		}
+	}
+	return layers, nil
+}
+
 func allocateLinuxResources(ctx context.Context, coi *createOptionsInternal, r *resources.Resources, isSandbox bool) error {
 	if coi.Spec.Root == nil {
 		coi.Spec.Root = &specs.Root{}
 	}
 	containerRootInUVM := r.ContainerRootInUVM()
 	if coi.Spec.Windows != nil && len(coi.Spec.Windows.LayerFolders) > 0 {
+		driverLayers, err := getContainerExtraLayers(coi.Spec.Annotations)
+		if err != nil {
+			return errors.Wrap(err, "failed to get driver layers for container")
+		}
+		coi.Spec.Windows.LayerFolders = append(driverLayers, coi.Spec.Windows.LayerFolders...)
+
+		log.G(ctx).WithField("layerdirs", coi.Spec.Windows.LayerFolders).Info("the new layer dirs with the container driver layers")
```

Comment:
Do we log the original layers anywhere else? I feel like the way this is worded we'd be able to view a log before the new layers were applied (or possibly just in the same log to make the comparison easier) if need be.

Reply:
do you feel like this is needed in particular? IMO the paths to the additional drivers will likely look much different than those of the normal layers. Plus we'll have the config logged already so we'd already know which were the extra layers paths.

Reply:
That's what I was asking :), I forget if we got rid of the config logging.

Reply:
I just wanted some way to see the original layers + whatever was extra but seems like we do